.github/workflows/docker.yml

```diff
@@ -30,14 +34,14 @@ jobs:
         id: buildx
         uses: docker/setup-buildx-action@v2
 
-      - name: Build and push to <insert repo>
+      - name: Build and push to quay.io/stevebs/heatcluster
         uses: docker/build-push-action@v3
         with:
           file: Dockerfile
           build-args: HEATCLUSTER_VER=${{ github.event.inputs.version }}
           push: true
           target: app
-          tags: heatcluster:${{ github.event.inputs.version }}
+          tags: quay.io/stevebs/heatcluster:${{ github.event.inputs.version }}
```

Comment:
See comment above (I think there is a typo here)

Reply:
Fixed this, too! Thanks